docs/sources/next/javascript-api/jslib/http-instrumentation-pyroscope/instrumenthttp.md

```diff
@@ -0,0 +1,50 @@
+---
+title: 'instrumentHTTP'
+description: 'instrumentHTTP instruments the k6 http module with baggage header.'
+weight: 01
+---
+
+# instrumentHTTP
+
+The `instrumentHTTP` function instruments the k6 http module with additional headers. It transparently replaces each of the k6 http module functions with versions that automatically attach a [baggage header](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/jslib/http-instrumentation-pyroscope#about-baggage-header)(by default) to every request.
+
+Instrumented functions include [del](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-http/del),[get](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-http/get),[head](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-http/head),[options](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-http/options),[patch](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-http/patch),[post](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-http/post),[put](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-http/head), and [request](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-http/request).
+
+This means that, to instrument the HTTP requests, you don't need to rewrite any code.
+Instead, call it once in the init context.
```

Comment:
In this sentence, "it" has an antecedent of "any code." however, the sentence reads as if you mean the module or the requests? 
```suggestion
Instead, call the module once in the init context.
```